File: Server/aoRss/Models/Db/RSSQuickClientModel.cs

using Contensive.Models.Db;
using System;

namespace Contensive.Addons.Rss.Models.Db {
    public class RSSClientModel : DbBaseModel {
        //
        //====================================================================================================
        /// <summary>
        /// table definition
        /// </summary>
        public static DbBaseTableMetadataModel tableMetadata { get; } = new DbBaseTableMetadataModel("RSS Clients", "ccRssClients", "default", false);
        // 
        // ====================================================================================================
        // -- instance properties
        // 
        public string url { get; set; }
        public int refreshhours { get; set; }
        public int numberOfStories { get; set; }
        // 
    }
}

File: Server/aoRss/Models/Db/RSSQuickClientModel.vb




Imports System
Imports System.Collections.Generic
Imports System.Text
Imports Contensive.BaseClasses

Namespace Models.Db
    Public Class RSSClientModel
        Inherits BaseModel
        Implements ICloneable
        '
        '====================================================================================================
        '-- const
        Public Const contentName As String = "RSS Clients"
        Public Const contentTableName As String = "ccRssClients"
        Private Shadows Const contentDataSource As String = "default"             '<------ set to datasource if not default
        '
        '====================================================================================================
        ' -- instance properties
        'instancePropertiesGoHere
        Public Property url As String
        Public Property refreshhours As Integer
        Public Property numberOfStories As Integer
        '
    End Class
End Namespace
